src/sentry/lang/javascript/processor.py

```diff
@@ -545,7 +545,11 @@ def preprocess_step(self, processing_task):
 
     def handles_frame(self, frame, stacktrace_info):
         platform = frame.get("platform") or self.data.get("platform")
-        return settings.SENTRY_SCRAPE_JAVASCRIPT_CONTEXT and platform in ("javascript", "node")
+        has_context_line = len(frame.get("context_line", "")) > 0
```

Comment:
`frame["context_line"]` can be None. I would do `has_context_line = bool(frame.get("content_line"))` instead.